components/kyma-environment-broker/common/orchestration/automock/K8sClientProvider.go

```diff
@@ -0,0 +1,38 @@
+// Code generated by mockery 2.9.0. DO NOT EDIT.
```

Comment:
This doesn't seem to be imported anywhere. Where is it used? upgrade tests seem to use github.com/kyma-project/control-plane/components/kyma-environment-broker/internal/orchestration/mock